karma.conf.js

```diff
@@ -44,6 +45,19 @@ module.exports = function(config) {
         },
         webpack: {
             mode: "none",
+            plugins: [
+                new webpack.ProvidePlugin({
+                    process: "process/browser"
+                })
+            ],
+            resolve: {
+                alias: {
+                    "../../../lib/linter$": "../../../build/eslint.js"
+                },
+                fallback: {
+                    path: "path-browserify"
+                }
```

Comment:
Same as above, we don't have this dependency now and this is probably unnecessary if we'll use `node-polyfill-webpack-plugin`?